Tools/YamlCreate.ps1

```diff
@@ -31,232 +31,663 @@ filter TrimString {
-    Write-Host "Tags: " -ForegroundColor Blue -NoNewline
-    Write-Host $Tags -ForegroundColor White
-}
-
-if (-not [string]::IsNullOrWhiteSpace($Description)) {
-    $string = "Description: $Description"
-    Write-Output $string | Out-File $ManifestName -Append
-    Write-Host "Description: " -ForegroundColor Blue -NoNewline
-    Write-Host $Description -ForegroundColor White
-}
-
-if (-not [string]::IsNullOrWhiteSpace($Homepage)) {
-    $string = "Homepage: $Homepage"
-    Write-Output $string | Out-File $ManifestName -Append
-    Write-Host "Homepage: " -ForegroundColor Blue -NoNewline
-    Write-Host $Homepage -ForegroundColor White
-}
-
-Write-Output "Installers:" | Out-File $ManifestName -Append
-
-$string = "  - Arch: $architecture"
-Write-Output $string | Out-File $ManifestName -Append
-Write-Host "Arch: " -ForegroundColor Blue -NoNewline
-Write-Host $architecture -ForegroundColor White
-
-$string = "    Url: $URL"
-Write-Output $string | Out-File $ManifestName -Append
-Write-Host "Url: " -ForegroundColor Blue -NoNewline
-Write-Host $URL -ForegroundColor White
-
-$string = "    Sha256: $Hash"
-Write-Output $string | Out-File $ManifestName -Append
-Write-Host "Sha256: " -ForegroundColor Blue -NoNewline
-Write-Host $Hash -ForegroundColor White
-
-$string = "    InstallerType: $InstallerType" 
-Write-Output $string | Out-File $ManifestName -Append
-Write-Host "InstallerType: " -ForegroundColor Blue -NoNewline
-Write-Host $InstallerType -ForegroundColor White
-
-if ((-not [string]::IsNullOrWhiteSpace($Silent)) -or 
-    (-not [string]::IsNullOrWhiteSpace($SilentWithProgress))) {
-    $string = "    Switches:"
-    Write-Output $string | Out-File $ManifestName -Append
-    Write-Host "Switches: " -ForegroundColor Blue -NoNewline
-}
-
-if (-not [string]::IsNullOrWhiteSpace($Silent)) {
-    $string = "      Silent: $Silent"
-    Write-Output $string | Out-File $ManifestName -Append
-    Write-Host "Silent: " -ForegroundColor Blue -NoNewline
-    Write-Host $Silent -ForegroundColor White
-}
-
-if (-not [string]::IsNullOrWhiteSpace($SilentWithProgress)) {
-    $string = "      SilentWithProgress: $SilentWithProgress"
-    Write-Output $string | Out-File $ManifestName -Append
-    Write-Host "SilentWithProgress: " -ForegroundColor Blue -NoNewline
-    Write-Host $SilentWithProgress -ForegroundColor White
-}
+# What does this stuff do??
 
-$ManifestsFolder = (Resolve-Path "$PSScriptRoot\..\manifests").Path
-$PublisherFolder = Join-Path $ManifestsFolder $Publisher
-$AppFolder = Join-Path $PublisherFolder $AppNameFolder
-New-Item -ItemType "Directory" -Force -Path $AppFolder | Out-Null
+#$FileOldEncoding = Get-Content -Raw $VersionManifest
+#Remove-Item -Path $VersionManifest
+#$ManifestPath = Join-Path $AppFolder $VersionManifest
+#$Utf8NoBomEncoding = New-Object System.Text.UTF8Encoding $False
+#[System.IO.File]::WriteAllLines($ManifestPath, $FileOldEncoding, $Utf8NoBomEncoding)
 
-$FileOldEncoding = Get-Content -Raw $ManifestName
```

Comment:
I checked with the original author (@KevinLaMS) and he said this was needed to ensure the output file was in the correct character set. I'd leave it in unless you can verify that without it it's still getting written as Utf8NoBomEncoding

Reply:
Aight, i've added it but i have no idea how to check if its correct